packages/odyssey-react-mui/src/Autocomplete.tsx

```diff
@@ -114,21 +130,26 @@ export type AutocompleteProps<
   /**
    * The options for the Autocomplete input
    */
-  options: MuiAutocompleteProps<
-    OptionType,
-    HasMultipleChoices,
-    undefined,
-    IsCustomValueAllowed
-  >["options"];
+  options: ReadonlyArray<OptionType>;
   /**
    * The value of the Autocomplete input
    */
-  value?: MuiAutocompleteProps<
+  value?: UseAutocompleteProps<
     OptionType,
     HasMultipleChoices,
     undefined,
     IsCustomValueAllowed
   >["value"];
+
+  /**
+   * Used to determine if the option represents the given value. Uses strict equality by default if none provided.
+   * Both arguments need to be handled, an option can only match with one value.
+   * option: the option to test
+   * value: the value to test against
+   *
+   * You will need to implement this function if your `option` items are objects.
+   */
+  isOptionEqualToValue?: (option: OptionType, value: OptionType) => boolean;
```

Comment:
Can we name this `getIsOptionEqualToValue`?
```suggestion
  /**
   * Used to determine if the option represents the given value. Uses strict equality by default if none provided.
   * Both arguments need to be handled, an option can only match with one value.
   * option: the option to test
   * value: the value to test against
   *
   * You will need to implement this function if your `option` items are objects.
   */
  getIsOptionEqualToValue?: (option: OptionType, value: OptionType) => boolean;
```

I ask because `is` and `has` represent booleans, and if you're getting a value via a function, it would be `getIs` or `getHas`. Sounds weird, but it's consistent and clear your intent.

Reply:
This is resolves to a boolean though; it is a comparator. I believe it is clear the way it currently stands and follows the way MUI has it as well. Unless you feel that strongly about it, I'd prefer to leave it as-is for the reasons I stated.

Reply:
I don't want to match with MUI because I don't like the way they name things. I'd like to change it to be consistent with the rest of Odyssey and the patterns we're telling people to follow.